pom.xml

```diff
@@ -796,9 +796,60 @@ under the License.</licenseText>
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.0</version>
+          <configuration>
+            <activeRecipes>
+              <!-- BestPractices -->
+              <!-- <recipe>org.openrewrite.java.recipes.JavaRecipeBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeNullabilityBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeTestingBestPractices</recipe> -->
+              <!-- testing -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.JUnitToAssertj</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.SimplifyAssertJAssertion</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.junit5.JUnit5BestPractices</recipe> -->
+              <!-- staticanalysis -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CodeCleanup</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CommonStaticAnalysis</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedLocalVariables</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateFields</recipe> -->
+              <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods</recipe>
+            </activeRecipes>
+            <exportDatatables>true</exportDatatables>
+            <failOnDryRunResults>true</failOnDryRunResults>
+            <exclusions>
+              <!-- wait for suppression: RemoveUnusedPrivateMethods -->
```

Comment:
need suppression to be exact not broad. @timtebeek

Reply:
its not working neither:

[WARNING] The recipe produced 17 warning(s). Please report this to the recipe author.
[WARNING] These recipes would make changes to src/mdo/java/WrapperProperties.java:
[WARNING]     org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods
[WARNING] These recipes would make changes to src/mdo/java/WrapperList.java:
[WARNING]     org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods